x/oracle/keeper/params_test.go

```diff
@@ -92,6 +92,35 @@ func TestUpdateTokenFeeder(t *testing.T) {
 			height: 1,
 			err:    types.ErrInvalidParams.Wrap("invalid tokenFeeder, non-exist tokenID referred"),
 		},
+		{
+			name: "invalid udpate, for existing feeder, feeder not started, and set endblock to history height",
+			tokenFeeder: types.TokenFeeder{
+				TokenID:  2,
+				EndBlock: 5,
+			},
+			height: 6,
+			err:    types.ErrInvalidParams.Wrapf("invalid tokenFeeder to update, invalid EndBlock, currentHeight: %d, set: %d", 6, 5),
+		},
+		{
+			name: "invalid update, for existing feeder, feeder started, and set endblock to history height including 0",
+			tokenFeeder: types.TokenFeeder{
+				TokenID: 2,
+			},
+			height: 11,
+			err:    types.ErrInvalidParams.Wrapf("invalid tokenFeeder to update, invalid EndBlock, currentHeight: %d, set: %d", 11, 0),
+		},
```

Comment:
add positive cases for startbaseblock and endblock plz.  also add negative cases when trying to update end_block when (endBlock-startBasedBlock)%interval<maxNonce. 

1. endBlock is a future block
2. endBlock > startBasedBlock
3. (endBlock-startBasedBlock)%interval≥maxNonce

Reply:
`(endblock-startBaseBlock) % interval < maxNonce` had been tested in this file('not set properly')